catalog/security/catalog-security-plugin/src/main/java/ddf/catalog/security/plugin/SecurityPlugin.java

```diff
@@ -44,20 +47,32 @@
 
     @Override
     public CreateRequest processPreCreate(CreateRequest input) throws StopProcessingException {
-        setSubject(input);
+        setSubjectOnRequestProperties(input);
+
+        Serializable subjectName = input.getProperties()
+                .get(SecurityConstants.SECURITY_SUBJECT);
+
+        if (input.getMetacards() != null && subjectName != null) {
+            input.getMetacards()
+                    .forEach(metacard -> {
+                        metacard.setAttribute(new AttributeImpl(Metacard.POINT_OF_CONTACT,
+                                SubjectUtils.getName((ddf.security.Subject) subjectName, "")));
```

Comment:
Removed the default. I had put that in there as the initial fix for easy testing, with the intent of going back and fixing it. Good catch.